app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class UserDisabled < StandardError
 
 class User < ActiveRecord::Base
   rolify
+  has_many :physical_tickets, through: :ticket_purchases, dependent: :destroy
```

Comment:
> so you can just add the by_conference scope here, and even remove it from the other model, as would only be used here. 

Sorry I didn't understand what you mean here. we already have by_conference scope in PhysicalTicket model why do we have to remove it from there and it in this user model. 😕